packages/connect/CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# 9.1.5
+
+-   feat(suite): T2B1 replace Model R name by official Trezor Safe 3 name (7460372ed1)
+-   feat(connect): check that CA certificate validity from is not in the future (249ddc358a)
+-   feat(connect): add production root and CA pubkeys (af907a296d)
+-   feat(connect): add authenticateDevice method (45b99c0813)
```

Comment:
please change these 3 lines 
```
-   feat(connect): check that CA certificate validity from is not in the future (249ddc358a)
-   feat(connect): add production root and CA pubkeys (af907a296d)
-   feat(connect): add authenticateDevice method (45b99c0813)
```
into 
```
- feat(connect): add authenticateDevice method (45b99c0813, af907a296d, 249ddc358a)
```

no need to mention small details in public changelog. we have git log for this